### PR TITLE
update guide from randomized to shuffle NFT drop

### PR DESCRIPTION
**Pull request by warengonzaga**
hey there!

- remove the old guide (Deploy a randomized NFT drop without writing any code)
- add a new version of the guide (How to shuffle your NFT drop without writing any code)
- completely from scratch guide from the latest UI/UX

@nachoiacovino kindly review
@rattrayd11 kindly prepare a new thumbnail for the guide

Cheers! 🎉

### CODE REVIEW

**Comment by warengonzaga** [comment]
Thanks, @rattrayd11 for the thumbnail! LFG!

File: guides/shuffle-nft-drop/index.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+authors: waren
+tags: [nft-drop, no-code, batch-upload, shuffle]
+image: "./thumbnail.png"
+date: "2022-07-26"
+---
+
+# How to shuffle your NFT drop without writing any code
```
Comment (nachoiacovino) [inline guides/shuffle-nft-drop/index.mdx:8]: Remove "How to"

File: guides/shuffle-nft-drop/index.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Learn how to shuffle your NFT drop without writing any code.
+
+<!-- truncate -->
+
+## Intro
+
+Let’s say you are ready to launch your first NFT project, but you want it to be shuffled for your community to mint. This is possible without writing any code by just using the thirdweb dashboard.
+
+In this guide, we’ll [deploy a collection using the NFT drop contract](https://portal.thirdweb.com/guides/release-an-nft-drop-with-no-code), after that, we'll **batch upload** it to the dashboard and shuffle the metadata arrangement using the dashboard only. I’ll assume you have the metadata and the image assets ready, if not it is fine, I would recommend watching Hashlips’ [Generate NFT art and metadata speed run](https://youtu.be/8PcncMYQRU0) video.
```
Comment (nachoiacovino) [inline guides/shuffle-nft-drop/index.mdx:18]: Internal links should not have portal.thirdweb.com in front, they should start with /. In this case /guides/release...

File: guides/shuffle-nft-drop/index.mdx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+## Setup
+
+Let’s assume you’ve already a MetaMask wallet, if not please follow our [create a MetaMask wallet tutorial](https://portal.thirdweb.com/guides/create-a-metamask-wallet).
```
Comment (nachoiacovino) [inline guides/shuffle-nft-drop/index.mdx:22]: Same thing here

File: guides/shuffle-nft-drop/index.mdx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+After that, click the checkbox for "**Shuffle the order of the NFTs before uploading.**" and click **upload**!
+
+![Shuffle checkbox](./shuffle-checkbox.png)
+
+:::caution Remember!
+
+- Shuffle feature is an off-chain operation and is not provable.
+- It will shuffle the order of **metadata** only **not the token ID**.
+- Minting will still follow the order of token ID.
```
Comment (nachoiacovino) [inline guides/shuffle-nft-drop/index.mdx:86]: "Minting will still be sequential by token ID"

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
Small things and let's ship it

File: guides/shuffle-nft-drop/index.mdx (hunk 2 above)
Comment (warengonzaga) [reply guides/shuffle-nft-drop/index.mdx]: replaced all the links. 💪

File: guides/shuffle-nft-drop/index.mdx (hunk 4 above)
Comment (warengonzaga) [reply guides/shuffle-nft-drop/index.mdx]: Done, thanks!

**Comment by warengonzaga** [comment]
Done with the requested changes @nachoiacovino 💪

**Review by nachoiacovino — APPROVED** [review]
lgtm!

**Pull request merged**